tests/integration-tests/tests/performance_tests/test_scaling.py

```diff
@@ -0,0 +1,55 @@
+import logging
+
+import pytest
+from remote_command_executor import RemoteCommandExecutor
+
+
+@pytest.mark.parametrize(
+    "max_nodes",
+    [1000],
+)
+def test_scaling(
+    vpc_stack,
+    instance,
+    os,
+    region,
+    scheduler,
+    pcluster_config_reader,
+    clusters_factory,
+    test_datadir,
+    scheduler_commands_factory,
+    max_nodes,
+):
+    cluster_config = pcluster_config_reader(max_nodes=max_nodes)
+    cluster = clusters_factory(cluster_config)
+
+    logging.info("Cluster Created")
+
+    remote_command_executor = RemoteCommandExecutor(cluster)
+    scheduler_commands = scheduler_commands_factory(remote_command_executor)
+
+    nodes_per_job = 2
+    num_jobs = int(max_nodes / nodes_per_job)
+
+    logging.info(f"Submitting {num_jobs} jobs on {max_nodes} dynamic nodes ({nodes_per_job} jobs/node)")
+    job_ids = []
+    for _ in range(0, num_jobs):
```

Comment:
have you considered using a job array rather than submitting multiple jobs? In this way you have a single job to monitor.

```
sbatch --wrap "sleep 100" --array 0-10 --exclusive
```

Reply:
Great point thank you! I'll do it